ui/pages/first-time-flow/welcome/welcome.test.js

```diff
@@ -45,15 +37,12 @@ describe('Welcome', () => {
       },
     };
 
-    const wrapper = mountWithRouter(
-      <Welcome.WrappedComponent {...props} />,
-      store,
-    );
+    renderWithProvider(<Welcome.WrappedComponent {...props} />);
 
-    const getStartedButton = wrapper.find(
-      '.btn-primary.first-time-flow__button',
-    );
-    getStartedButton.simulate('click');
+    const getStartedButton = screen.getByTestId('first-time-flow__button');
+
+    // getStartedButton.simulate('click');
```

Comment:
```suggestion
```

Reply:
To confirm, we are keeping this comment?

Reply:
To confirm, we are keeping this comment? @tmashuang

Reply:
Hmm I thought this got resolved in [9624b17](https://github.com/MetaMask/metamask-extension/pull/15730/commits/9624b17d190c68f9ef77cff0ac97cce8c6e6198e), will resolve properly.